src/app/js/admin/fields/FieldForm.js

```diff
@@ -14,8 +14,7 @@ import {
     saveField,
 } from './';
 
-import { fromFields } from '../../selectors';
-
+import { fromFields } from '../';
```

Comment:
What do we import here?
When we import an index we sort of import a directory, and a directory should not be imported from its children IMO. avoid all from '../'